Lib/defcon/objects/contour.py

```diff
@@ -715,8 +715,8 @@ class Recorder(object):
 
         Method calls to be recorded must not start with an underscore.
 
-        This class defines a public method with a private(!) attribute name:
-        "Recorder._play(self, target)" because that way calls to all methods
+        This class creates a callable object which can be used like a
+        function: "recorder(target)" because that way calls to all methods
```

Comment:
I think here it should read like this:
```
This class creates a callable object which can be used like a
function, e.g. `recorder(target)`.
```

the following phrase "because that way..." is confusing to me.